From: Andreas Stieger <andreas.stieger@gmx.de>
Date: Wed, 30 Apr 2014 20:30:57 +0100
Subject: remove __DATE__ and __TIME__
References:
Upstream: never

Fixes warning and prevents unneccessary rebuilds:

LiE.i586: W: file-contains-date-and-time /usr/lib/LiE/lie
Your file uses  __DATE and __TIME__ this causes the package to rebuild when
not needed

---
 date.c |    2 +-
 init.c |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

Index: LiE/date.c
===================================================================
--- LiE.orig/date.c	1996-08-30 13:05:15.000000000 +0100
+++ LiE/date.c	2014-04-30 20:28:04.000000000 +0100
@@ -1 +1 @@
-char date[] = __DATE__ " at " __TIME__;
+char date[] = "";
Index: LiE/init.c
===================================================================
--- LiE.orig/init.c	2014-04-30 20:28:14.000000000 +0100
+++ LiE/init.c	2014-04-30 20:28:43.000000000 +0100
@@ -150,8 +150,8 @@ void init(void)
   initialize_readline();
   
   { if (!redirected_input)
-    { printf("\nLiE version 2.2.2 created on %s\n" 
-        "Authors: Arjeh M. Cohen, Marc van Leeuwen, Bert Lisser.\n", date);
+    { printf("\nLiE version 2.2.2\n" 
+        "Authors: Arjeh M. Cohen, Marc van Leeuwen, Bert Lisser.\n");
       { FILE* f= fopen(titlefil,"r");
         if (f==NULL) printf("Purpose: development CWI\n\n");
         else { while ((ch=getc(f)) != EOF) putchar(ch); fclose(f); }
